seafowl.toml

```diff
@@ -15,4 +15,4 @@ bind_port = 6432
 bind_host = "0.0.0.0"
 
 [frontend.flight]
-bind_host = "127.0.0.1"
+bind_host = "0.0.0.0"
```

Comment:
Can you do it as an envvar in the Dockerfile instead? Seeing 0.0.0.0 anywhere, even in a default dev config file, gets me worried :D